tests/regression/nextcloud-rule-exclusions-plugin/9508721.yaml

```
@@ -0,0 +1,80 @@
+              of smoked paprika.\n\nIf you're not sure how best to cut and prep your pumpkin (be it butternut squash or another type of pumpkin) for your soup, check out our video guide at the bottom of
+              the method steps.\n\nWe also have a flavourful vegan pumpkin soup recipe for those who prefer a dairy-free soup. Looking for more easy soup recipe ideas? Try our 15-minute chicken and corn noodle soup.
+              You could also make pumpkin soup in a slow cooker.","url":"https://www.taste.com.au/recipes/pumpkin-soup/e54ab2b5-7d19-4e4b-b383-781292dc4bfb?r=quickeasy/biccuul7",
+              "image":"/best-easy-pumpkin-soup-recipe-185570-1.jpg","prepTime":"PT0H10M0S","cookTime":"PT0H40M0S","totalTime":"PT0H50M0S","recipeCategory":"Soup","keywords":"Pumpkin,Soup,Easy","recipeYield":6,
+              "tool":[],"recipeIngredient":["1L Massel chicken style liquid stock or vegetable liquid stock","1 large potato, peeled, diced","1kg peeled pumpkin, diced","1/2 tsp freshly grated nutmeg",
+              "1 tsp ground cumin","1/2 tsp ground coriander","1 garlic clove, crushed","1 leek, white part only, finely sliced","1 onion, finely chopped","2 tbsp Coles Classic Olive Oil"],
+              "recipeInstructions":["Heat oil in a large saucepan over low heat, add onion and leek and cook for 2-3 minutes, until softened but not coloured.","Add garlic, coriander, cumin, and nutmeg and cook,
+              stirring, for 30 seconds. Add pumpkin, potato and stock and bring to the boil. Turn heat to low, cover and simmer for 30 minutes. Allow to cool slightly, then blend in batches.",
+              "Return soup to pan, stir through cream and reheat gently. Season and add a little more nutmeg if desired."],"nutrition":{"calories":"206cal","proteinContent":"3.1g","fatContent":"12.9g",
+              "saturatedFatContent":"5.0g","carbohydrateContent":"21.9g","sugarContent":"5.8g","sodiumContent":"563.8mg","@type":"NutritionInformation"},"valueInit":{"id":0,"name":"","description":"","url":"",
+              "image":"","prepTime":"","cookTime":"","totalTime":"","recipeCategory":"","keywords":"","recipeYield":"","tool":[],"recipeIngredient":[],"recipeInstructions":[],"nutrition":[]},
+              "@context":"http://schema.org","@type":"Recipe","dateCreated":"2024-07-22T12:29:24+00:00","dateModified":"2024-07-22T12:29:24+00:00","datePublished":null,"printImage":true,
+              "imageUrl":"/apps/cookbook/webapp/recipes/1245809/image?size=full"},"@context":"http://schema.org","@type":"Recipe","dateCreated":"2024-07-22T12:29:24+00:00",
+              "dateModified":"2024-07-22T12:29:24+00:00","datePublished":null,"printImage":true,"imageUrl":"/apps/cookbook/webapp/recipes/1245809/image?size=full"}
+            version: HTTP/1.1
+          output:
+            no_log_contains: |
```

Careful here, without the chomping indicator, the regular expression will contain a line break at the end.
```suggestion
            no_log_contains: |-
```